content/en/user-guide/aws/backup/index.md

```diff
@@ -1,11 +1,131 @@
 ---
 title: "Backup"
 linkTitle: "Backup"
-categories: ["LocalStack Pro"]
-description: >
-  Get started with AWS Backup on LocalStack
-aliases:
-  - /aws/backup/
+description: Get started with Backup on LocalStack
 ---
 
-The [Backup API](https://docs.aws.amazon.com/aws-backup/) allows to manage backup plans, to create scheduled or on-demand backups of certain resource types like DynamoDB tables or RDS databases.
+## Introduction
+
+Backup is a centralized backup service provided by Amazon Web Services. It simplifies the process of backing up and restoring your data across various AWS services and resources. Backup supports a wide range of AWS resources, including Elastic Block Store (EBS) volumes, Relational Database Service (RDS) databases, DynamoDB tables, Elastic File System (EFS) file systems, and more. Backup enables you to set backup retention policies, allowing you to specify how long you want to retain your backup copies.
+
+LocalStack supports Backup via the Pro/Team offering, allowing you to use the Backup APIs in your local environment to manage backup plans, create scheduled or on-demand backups of certain resource types. The supported APIs are available on our [API coverage page](https://docs.localstack.cloud/references/coverage/coverage_backup/), which provides information on the extent of Backup's integration with LocalStack.
+
+## Getting started
+
+This guide is designed for users new to Backup and assumes basic knowledge of the AWS CLI and our [`awslocal`](https://github.com/localstack/awscli-local) wrapper script.
+
+Start your LocalStack container using your preferred method. We will demonstrate how to create a backup job and specify a set of resources to the backup plan name and backup rules with the AWS CLI.
+
+### Create a backup vault
+
+You can create a backup vault which acts as a logical container where backups are stored using the [`CreateBackupVault`](https://docs.aws.amazon.com/aws-backup/latest/devguide/API_CreateBackupVault.html) API. Run the following command to create a backup vault named `my-vault`:
+
+{{< command >}}
+$ awslocal backup create-backup-vault \
+    --backup-vault-name primary
+{{< / command >}}
+
+The following output would be retrieved:
+
+```bash
+{
+    "BackupVaultName": "primary",
+    "BackupVaultArn": "arn:aws:backup:us-east-1:000000000000:backup-vault:primary",
+    "CreationDate": 1693286432.432258
+}
+```
+
+### Create a backup plan
+
+You can create a backup plan which specifies the resources to backup, the backup vault to store the backups in, and the schedule for creating backups. You can specify the backup plan in a `backup-plan.json` file:
```

Comment:
q: I'm not seeing the "specifies the resources to backup" for the backup plan, only for the backup selection which is linked to a backup plan. Isn't the backup plan mostly for the backup scheduling and lifecycle with `CopyActions`?